docs/toc.md

```diff
@@ -57,6 +57,7 @@
 ## [Get started](core/get-started.md)
 ## [Windows Prerequisites](core/windows-prerequisites.md)
```

Comment:
wondering if we should move these prereqs topics under Get Started in the TOC. That is, just add an extra #.